pkg/core/consensus/user/sortition.go

```diff
@@ -150,7 +150,7 @@ func (p Provisioners) CreateVotingCommittee(seed []byte, round uint64, step uint
 
 		// Deduct up to 1 DUSK from the extracted member's stake.
 		m := p.GetMember(blsPk)
-		subtracted := m.SubtractFromStake(1 * config.DUSK)
+		subtracted := m.SubtractFromStake(10)
```

Comment:
What is the meaning of value 10 here? The comment  says `deduct up to 1 DUSK`.

Reply:
If I'm not mistaken, we substract `1_000_000_000` in the rust implementation?
FYI @herr-seppia

Reply:
You're right, good catch

This is a leftover of some tests

Reply:
Restored to 1 Dusk